microshift_running_apps/microshift-embed-apps-offline-use.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="microshift-embed-apps-offline-use"]
+= Embedding {product-title} applications for offline use
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-embed-apps-offline-use
+
+toc::[]
+
+You can embed microservices-based workloads and applications in a {op-system-ostree-first} image. This allows users to run a {product-title} cluster in air-gapped, disconnected, or offline environment.
+
+include::modules/microshift-embed-images-offline-use.adoc[leveloffset=+1]
+
+//additional resources for embedding images for offline use
+[id="Additional-resources_microshift-embed-apps-offline-use_{context}"]
```

Comment:
The `id` is only for assembly additional resources sections with a heading. Trailing includes just have the `role`.